site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -177,6 +177,125 @@ spec:
+
+```yaml
+apiVersion: autoscaling.agones.dev/v1
+kind: FleetAutoscaler
+metadata:
+  name: schedule-fleet-autoscaler
+spec:
+  fleetName: fleet-example
+  policy:
+    # Schedule based policy for autoscaling.
+    type: Schedule
+    schedule:
+      between:
+        # The policy becomes eligible for application starting on October 31st, 2024 at 12:00 AM PST. If not set, the policy will immediately be eligible for application.
+        start: "2024-10-31T00:00:00-07:00"
+        # The policy is never ineligible for application. If not set, the policy will always be eligible for application (after the start time).
+        end: ""
+      activePeriod:
+        # Use PST time for the startCron field. Defaults to UTC if not set.
+        timezone: "America/Los_Angeles"
+        # Start applying the policy everyday at 12:00 AM PST. If not set, the policy will always be applied in the .between window.
+        # (Only eligible starting on October 31, 2024 at 12:00 AM PST).
+        startCron: "0 0 * * *"
+        # Apply this policy indefinitely. If not set, the duration will be defaulted to always/indefinite.
+        duration: ""
+      # Policy to be applied during the activePeriod. Required.
+      policy:
+        type: Buffer
+        buffer:
+          bufferSize: 5
+          minReplicas: 10
+          maxReplicas: 20
+  # The autoscaling sync strategy, this will determine how frequent the schedule is evaluated.
+  sync:
+    # type of the sync. for now, only FixedInterval is available
+    type: FixedInterval
+    # parameters of the fixedInterval sync
+    fixedInterval:
+      # the time in seconds between each auto scaling
+      seconds: 30
+```
+
+A Chain based autoscaler can be used to autoscale `GameServers` based on a list of policies, the index of each policy within the list determines the priority in which the policies are evaluated and applied.
+
+For example, if you have an in-game event happening between 12:00 AM and 10:00 PM on October 31, 2024 EST, and you want to ensure there are always at least 5 ready game servers during that time, you chain add a schedule policy that applies a buffer policy with a buffer size of 5. Now, lets say you want to adjust the number of game servers after the event based on external factors after the in-game event you could chain a webhook policy to dynamically. Finally as a fallback, lets say webhook invocation fails you can chain another buffer policy to default to.
+
+{{< alert title="Note" color="info">}}
+While it's possible to use multiple FleetAutoscalers to perform the same actions on a fleet, this approach can result in unpredictable scaling behavior and overwhelm the controller. Multiple FleetAutoscalers pointing to the same fleet can lead to conflicting schedules and complex management. To avoid these issues, it's recommended to use the Chain Policy for defining scheduled scaling. This simplifies management and prevents unexpected scaling fluctuations.
+{{< /alert >}}
```

Comment:
I would move this note up to just after the scheduled autoscaler example. That way the chain autoscaler explanation is right next to the associated example.